TypeCobol/Compiler/Diagnostics/CrossChecker.cs

```diff
@@ -746,13 +747,68 @@ public override bool Visit(DataDefinition dataDefinition)
                         "Group item " + dataDefinition.Name + " cannot have \"Blank when zero\" clause",
                         dataDefinitionEntry);
                 }
+            }
+            else if (commonDataDataDefinitionCodeElement != null) 
+            {
+                //This is an elementary item. It must have a picture, type or a valid usage
+                var levelNumberValue = levelNumber?.Value;
+                if ((levelNumberValue < 50 || levelNumberValue == 77)
+                    && dataDefinition.Picture == null
+                    //commonDataDataDefinitionCodeElement.UserDefinedDataType is the type as written in the code. dataDefinition.TypeDefinition is the resolved type which can be null if the type cannot be found
+                    && commonDataDataDefinitionCodeElement.UserDefinedDataType == null 
+                    && (!dataDefinition.Usage.HasValue || !IsUsageAllowedWithoutPicture(dataDefinition.Usage.Value)))
+                {
+                    DiagnosticUtils.AddError(dataDefinition, "A group item cannot be empty. Add children, picture or valid usage (INDEX, COMP-1, COMP-2, POINTER, POINTER-32, PROCEDURE-POINTER or FUNCTION-POINTER).", commonDataDataDefinitionCodeElement);
 
-                return true;
+                    //Detect copy included at wrong level (because there is already the same level in the copy)
+                    CheckCopyAtWrongLevel(dataDefinition, commonDataDataDefinitionCodeElement);
+                }
+            }
+
+            if (commonDataDataDefinitionCodeElement?.Usage != null &&
```

Comment:
Style: put `commonDataDataDefinitionCodeElement?.Usage` or `commonDataDataDefinitionCodeElement?.Usage?.Value` in a variable to avoid long line.

Reply:
Done